analyzers/its/expected/Net5/Net5--net5.0-S2952.json

```diff
@@ -1,30 +0,0 @@
-{
-"issues":  [
-{
-"id":  "S2952",
-"message":  "Move this 'Dispose' call into this class' own 'Dispose' method.",
-"location":  {
-"uri":  "sources\Net5\Net5\S2952.Part1.cs",
-"region":  {
-"startLine":  22,
-"startColumn":  21,
-"endLine":  22,
-"endColumn":  28
```

Comment:
True Negativ. See also the fixes in the test cases.
https://github.com/SonarSource/sonar-dotnet/blob/9164a51158b1991afa96452f56106d51fb453cb5/analyzers/its/sources/Net5/Net5/S2952.Part1.cs#L22